WalletWasabi.Fluent/ViewModels/Wallets/WalletViewModel.cs

```diff
@@ -57,12 +57,12 @@ protected WalletViewModel(Wallet wallet) : base(wallet)
 					nameof(Wallet.TransactionProcessor.WalletRelevantTransactionProcessed))
 				.Select(_ => Unit.Default)
 				.Merge(Observable.FromEventPattern(Wallet, nameof(Wallet.NewFilterProcessed))
-					.Select(_ => Unit.Default))
+						.Select(_ => Unit.Default))
 				.Merge(Services.UiConfig.WhenAnyValue(x => x.PrivacyMode).Select(_ => Unit.Default))
 				.Merge(Wallet.Synchronizer.WhenAnyValue(x => x.UsdExchangeRate).Select(_ => Unit.Default))
 				.Merge(CoinJoinSettings.WhenAnyValue(x => x.AnonScoreTarget).Select(_ => Unit.Default).Skip(1).Throttle(TimeSpan.FromMilliseconds(3000))
-				.Throttle(TimeSpan.FromSeconds(0.1))
-				.ObserveOn(RxApp.MainThreadScheduler));
+					.Throttle(TimeSpan.FromSeconds(0.1)))
```

Comment:
A small nit: why not put this at the end of LOC 63 to avoid such mistakes or any confusions.